common/test/acceptance/tests/lms/test_teams.py

```diff
@@ -801,21 +800,21 @@ def test_search(self):
         self.create_teams(self.topic, 5)
         self.browse_teams_page.visit()
         events = [{
-            'event_type': 'edx.team.searched',
+            'event_type': 'edx.team.page_viewed',
             'event': {
-                'search_text': search_text,
+                'page_name': 'search-teams',
                 'topic_id': self.topic['id'],
-                'number_of_results': 0
+                'team_id': None
             }
         }, {
-            'event_type': 'edx.team.page_viewed',
+            'event_type': 'edx.team.searched',
             'event': {
-                'page_name': 'search-teams',
+                'search_text': search_text,
                 'topic_id': self.topic['id'],
-                'team_id': None
+                'number_of_results': 0
             }
         }]
-        with self.assert_events_match_during(self.only_team_events, expected_events=events):
+        with self.assert_events_match_during(self.only_team_events, expected_events=events, in_order=False):
```

Comment:
Seems fair. There's not really any reason against doing this, so if it makes the tests not be flaky I'm all for it.